wwwroot/init/test.json

```diff
@@ -1,278 +1,355 @@
+              "tableStyle": {
+                "dataVariable": "ITU-T Telephone Code"
+              }
+            },
+            {
+              "name": "Droughts by Country",
+              "type": "csv",
+              "url": "test/droughts.csv",
+              "tableStyle": {
+                "featureInfoFields": {
+                  "Country name": "Country",
+                  "date": "Date",
+                  "Total deaths": "Deaths",
+                  "Total damage": "Cost"
                 },
+                "dataVariable": "Total affected",
+                "displayDuration": 525600
+              }
+            },
+            {
+              "name": "State Mask Test",
+              "type": "csv",
+              "url": "test/states_stem_2013.csv",
+              "tableStyle": {
+                "colorMap": [
+                  {
+                    "color": "rgba(0,0,0,0.0)",
+                    "offset": 0
+                  },
+                  {
+                    "color": "rgba(0,0,0,1.0)",
+                    "offset": 1
+                  }
+                ],
+                "dataVariable": "Mask"
+              },
+              "keepOnTop": true,
+              "opacity": 1
+            }
+          ]
+        },
+        {
+          "name": "KML/KMZ files",
+          "type": "group",
+          "items": [
+            {
+              "name": "Simple Test (KML)",
+              "type": "kml",
+              "url": "test/simple.kml"
+            },
+            {
+              "name": "Vic Police (KML)",
+              "type": "kml",
+              "url": "test/vic_police.kml"
+            },
+            {
+              "name": "Vic Police (KMZ)",
+              "type": "kml",
+              "url": "test/vic_police.kmz"
+            }
+          ]
+        },
+        {
+          "name": "CZML files",
+          "type": "group",
+          "items": [
+            {
+              "name": "Simple Test (CZML)",
+              "type": "czml",
+              "url": "test/simple.czml"
+            },
+            {
+              "name": "Large feature info box (CZML)",
+              "type": "czml",
+              "url": "test/bigInfobox.czml"
+            },
+            {
+              "name": "Embedded CZML",
+              "type": "czml",
+              "data": [
                 {
-                    "name": "data.gov.au",
-                    "url": "http://data.gov.au/geoserver/ows",
-                    "type": "wfs-getCapabilities"
+                  "id": "document",
+                  "version": "1.0"
                 },
                 {
-                    "name": "Western Australia",
-                    "url": "http://www2.landgate.wa.gov.au/ows/wfspublic_4326/wfs",
-                    "type": "wfs-getCapabilities"
+                  "position": {
+                    "cartographicDegrees": [
+                      133,
+                      -25,
+                      ""
```

Comment:
This catalog item doesn't work anymore because zeros were replaced with empty strings.